projects/storefrontstyles/scss/components/store-finder/_store-finder-list-item.scss

```diff
@@ -23,11 +20,34 @@
 
   .cx-store-name {
     @include type('6');
-    cursor: pointer;
+    cursor: default;
     min-height: 40px;
+    button {
+      text-align: left;
```

Comment:
you could use `button-reset()` mixin here, so you don't have to override the CSS manually here

Reply:
Replaced code by mixin.